pkg/apis/proto/function/v1/udfunction.proto

```diff
@@ -11,6 +11,9 @@ service UserDefinedFunction {
   // MapFn applies a function to each datum request element.
   rpc MapFn(DatumRequest) returns (DatumResponseList);
 
+  // MapStreamFn applies a function to each datum element and returns a stream.
+  rpc MapStreamFn(DatumRequest) returns (stream DatumResponseList);
```

Comment:
Is this right? Should it be `returns (stream DatumResponse)`?

Reply:
Put DatumResponseList in case users want to return multiple DatumResponse at a time. But that doesn't really matter as we are streaming.